io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,121 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "missioncenter",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=org.gnome.Settings",
+    "--system-talk-name=org.freedesktop.systemd1",
+    "--system-talk-name=org.freedesktop.NetworkManager",
+    "--filesystem=xdg-data/flatpak/exports/share:ro",
+    "--filesystem=/var/lib/flatpak/exports/share:ro",
+    "--filesystem=xdg-data/flatpak/app:ro",
+    "--filesystem=/var/lib/flatpak/app:ro",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share"
+  ],
+  "build-options": {
+    "append-path": "/app/tmp/rust/cargo/bin",
+    "build-args": [
+      "--share=network"
```

Comment:
this is not allowed. build must be offline.

Reply:
OK, I'll see what I can do

Reply:
How is this normally done with a Rust project that uses `crates.io` for its dependencies?

Reply:
if you don't have crates vendoring in a tarball, https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo

Reply:
Updated to fetch everything as a module source